src/shared/components/card/cardProduct/CardProduct.tsx

```
@@ -60,20 +140,98 @@ const CardProduct = ({
             </LinkButton>
           )}
         </div>
+
+        {isLarge && (
+          <div
+            className={styles.saveBtnOverlay}
+            onClick={(event) => event.stopPropagation()}
+            onKeyDown={(event) => event.stopPropagation()}
+          >
+            <SaveButton
+              disabled={disabled}
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)